client/my-sites/earn/home.tsx

```diff
@@ -93,7 +94,7 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 			'https://wordpress.com/support/wordpress-editor/blocks/simple-payments-block/';
 		const cta = hasSimplePayments
 			? {
-					text: translate( 'Collect one-time payments' ),
+					text: translate( 'Add one-time payments' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Collect One-time Payments' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/earn-page-copy-updates)